### DISCUSSION TITLE
reading writing comments in wav-file

### DISCUSSION THREAD

**Discussion by troglobytor** [audacity/audacity#2855] [Q&A]
I am trying to read comments made in Audacity with my own code and I have not found how in Audacity source yet. Any info on where to look very appreciated. I was writing in comments the text for spoken data, and was wanting to retrieve it to display with played sound.

**Comment by LWinterberg** [comment]
The labels in Audacity are not exported into the file when you export audio, instead you need to export them separately (File > Export > Export Labels)

**Reply by troglobytor** [reply]
I am referring to metadata in WAV-RIFF-AVI file which Audacity offers a dialogue to set when exporting as wav-file, specifically the "comment" section. It is with data like "Artist". Tag I believe is somewhere in "LIST" with value "ICMT".
